### DISCUSSION TITLE
Class name obfuscation in tailwindui

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#13617] [Help]
Looking for a way to obfuscate class names. I was thinking of using module CSS, but with tailwindcss, it seems no good. There are very few resources available on this topic, and none seem to be very helpful to me. I'm curious about how the tailwindui team was able to obfuscate class names that require a license. Do you have any idea about this? It would be great if you could help me out.

![image](https://github.com/tailwindlabs/tailwindcss/assets/134765302/658e8484-605b-4c40-b97d-b9ef3c16aa9b)

**Comment by Mardoxx** [comment]
Also looking for this.
